### ISSUE TITLE
What does this mean in my logs?

### ISSUE THREAD

**Issue by BobWs** [dani-garcia/bitwarden_rs#442]
What does this mean? I'm seeing it a lot in my logs:

> 2019-03-25 12:23:51,stdout,[2019-03-25 12:23:51][tokio_reactor][DEBUG] dropping I/O source: 1
> 
> 2019-03-25 12:23:51,stdout,[2019-03-25 12:23:51][tokio_reactor::registration][DEBUG] scheduling Read for: 1
> 
> 2019-03-25 12:23:51,stdout,[2019-03-25 12:23:51][tokio_reactor][DEBUG] dropping I/O source: 0
> 
> 2019-03-25 12:23:51,stdout,[2019-03-25 12:23:51][tokio_reactor::registration][DEBUG] scheduling Read for: 0
> 
> 2019-03-25 12:23:51,stdout,[2019-03-25 12:23:51][tokio_reactor][DEBUG] dropping I/O source: 4
> 
> 2019-03-25 12:23:51,stdout,[2019-03-25 12:23:51][tokio_reactor::registration][DEBUG] scheduling Read for: 4
> 
> 2019-03-25 12:23:51,stdout,[2019-03-25 12:23:51][tokio_reactor][DEBUG] dropping I/O source: 2
> 
> 2019-03-25 12:23:51,stdout,[2019-03-25 12:23:51][tokio_reactor::registration][DEBUG] scheduling Read for: 2
> 
> 2019-03-25 12:23:51,stdout,[2019-03-25 12:23:51][tokio_reactor][DEBUG] dropping I/O source: 3
> 
> 2019-03-25 12:23:51,stdout,[2019-03-25 12:23:51][tokio_reactor::registration][DEBUG] scheduling Read for: 3
> 
> 2019-03-25 12:23:27,stdout,[2019-03-25 12:23:27][tokio_reactor][DEBUG] dropping I/O source: 8
> 
> 2019-03-25 12:23:16,stdout,[2019-03-25 12:23:16][_][INFO] Response succeeded.
> 
> 2019-03-25 12:23:16,stdout,[2019-03-25 12:23:16][_][INFO] Outcome: Success

**Comment by BlackDex** [comment]
Seems you are running a "debug" version instead of the "release" version.
It's just normal output.

**Comment by BobWs** [comment]
That is strange I'm using the latest image "mprasil/bitwarden:latest"

**Comment by dani-garcia** [comment]
That's from the reqwest library that we use for icon downloading which uses the tokio crate, we currently hardcode the log level to debug globally (except a couple of dependencies which are set to a higher value), so this is what we get. It should happen regardless of what version you are using.

We don't really use the debug log level ourselves, so I've changed it to Info by default, which makes more sense and would hide that kind of message (which isn't very useful for us anyway). I also implemented a log_level config option. This should be available in the latest commit now, and in the built images soon.
